### DISCUSSION TITLE
질문이 있습니다

### DISCUSSION THREAD

**Discussion by lunapsy** [bosskmk/pluto_grid#541] [Q&A]
PlutoColumn에 frozen: PlutoColumnFrozen.none 옵션을 주어도

시작에 고정, 끝에 고정 메뉴가 활성화되지 않습니다. 어떻게 해야하나요?

**Comment by bosskmk** [comment]
컬럼 우측 메뉴에 있는 시작에 고정, 끝에 고정 메뉴가 나타난 상태에서
비활성화(클릭 불가) 상태라는 말씀이신거죠?

컬럼 우측 메뉴의 고정 메뉴는 넓이 제약 조건에 따라 비활성화 됩니다.
가운데 비고정 컬럼영역의 넓이가 200px 보다 작으면 비고정 컬럼을 고정 컬럼으로 이동 할 수 없습니다.

예를 들어 전체 그리드 넓이가 1000이고 현재 고정 된 컬럼들의 넓이 합계가 800을 넘어서면
비고정 컬럼을 고정 컬럼으로 이동 할 수 없습니다.(컬럼 우측 메뉴의 고정 메뉴들도 비활성화됨)
반대로 고정 컬럼을 비고정 컬럼으로는 이동 할 수 있습니다.

**Reply by lunapsy** [reply]
답변 감사드립니다.
